tests/libtransmission/net-test.cc

```diff
@@ -180,8 +180,92 @@ TEST_F(NetTest, isGlobalUnicastAddress)
     }
 }
 
-TEST_F(NetTest, globalIPv6)
+namespace libtransmission::test
 {
-    auto const addr = tr_globalIPv6();
-    EXPECT_TRUE(!addr || addr->is_global_unicast_address());
+
+using GlobalIPTest = SessionTest;
```

Comment:
If you agree with the idea of decoupling `tr_global_ip_cache` from `tr_session`, then this wouldn't need a session in order to be tested